### ISSUE TITLE
[Adaptive Cards> Blogs> Community Call - March 12, 2020]: Ensures <img> elements have alternate text or a role of none or presentation.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8218]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25267.1000)
Browser: Edge Dev (110.0.1556.0 (Official build) dev (64-bit)
URL: https://adaptivecards.io/blog/2020/Community-Call-March/
Tool: Accessibility Insights for Web

**Repro Steps:**

1. Open the above URL in edge dev browser.
2. Run the fast pass of Accessibility Insights for Web
3. Observe the issue.

**Actual Result:**
Element does not have an alt attribute or aria-label attribute does not exist or is empty.

**Expected Result:**
Ensures <img> elements have alternate text or a role of none or presentation i.e proper alt should be defined for the image.

**User Impact:**
Screen reader user will have difficulty in interacting with the contents and in understanding the purpose of the image if alt is not defined.

**MAS Reference link**:
https://microsoft.sharepoint.com/:w:/s/accessibility/EUEQyqOohQJFi5arJCWo8lcBzI00kLMvgq9T8QC1ygyhlg?e=ITurc7

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

<img width="953" alt="MAS1 1 1 Alt is not defined for the image present under blog" src="https://user-images.githubusercontent.com/104483005/211272696-faa3245d-ae73-45bd-91f4-1fc3e6051c6d.png">
<img width="960" alt="MAS1 1 1 Alt is not defined for the image present under Blog  community" src="https://user-images.githubusercontent.com/104483005/211275048-149257c3-5fdb-4793-9144-425a0349e578.png">

### Card JSON

```JSON
NA
```

**Comment by paulcam206** [comment]
@anna-dingler care to take a look?

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/14429f2e-a4cd-4c6f-9619-54363034c117)
